templates/provider/projectsveltos/Chart.yaml

```diff
@@ -2,9 +2,12 @@ apiVersion: v2
 name: projectsveltos
 description: A Helm chart to refer the official projectsveltos helm chart
 type: application
-version: 0.49.0
-appVersion: "0.49.0"
+version: 0.49.1
+appVersion: 0.49.0
```

Comment:
Shouldn't we bump the `appVersion` as well?

Reply:
Bumped both chart version and the application version to the `0.50.0`